lessons/probability.ipynb

```diff
@@ -221,6 +221,7 @@
    },
    "source": [
     "$P(Y|X)$ is a (normalized) slice through $P(X,Y)$ rather than an integral.\n",
+    "> _In the lefthand panel, imagine a vertical cut at $x=2$, to give us $P(y|x=2)$ in the righthand panel._\n",
```

Comment:
I'd prefer to issue this as a figure improvement (show the cut at x=2).
```suggestion
```

Reply:
Fine with me.